### ISSUE TITLE
[QUIC] Test .NET 7.0 on MsQuic 2.2

### ISSUE THREAD

**Issue by CarnaViire** [dotnet/runtime#83537]
After MsQuic 2.2 will be released, we need to verify that updating from 2.1 to 2.2 will go seamlessly for all QUIC-supported platforms on .NET 7.0.
- Testing Linux is a must-have (so users updating to latest-version libmsquic would not have any problems)
- Testing Windows is optional (but interesting from future servicing perspective)

**Comment by wfurt** [comment]
It seems like some test images already pick up the updates 
```
docker run -ri mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-36-helix bash
[helixbot@f2165bc199b6 /]$ rpm -qa| grep msquic
libmsquic-2.2.0-1.x86_64
libmsquic-1.9.1-1.x86_64
```

I did not check all platforms but I would expect images that use package manager to converge on new version as they get rebuild. (AFAK full Helix VMs would also pick up updated on regular bases) 


I'm not sure if there is anything else left @CarnaViire. I'm not sure if we should update Windows to 2.2 as it is marked as pre-release

**Comment by CarnaViire** [comment]
7.0 release branch is running on Fedora 34 https://helixre107v0xd1eu3ibi6ka.blob.core.windows.net/dotnet-runtime-refs-heads-release-70-staging-5425e256aa714e7480/System.Net.Quic.Functional.Tests/1/console.7d4ef317.log?helixlogtype=result 😑 I'll verify stuff manually later.

Windows part was not about updating right now, but to verify it works so in case of servicing we can just pull in the latest MsQuic instead of backporting a change. But it is just nice-to-have

**Comment by ManickaP** [comment]
We have already released 7.0 which depends on msquic 2.2.2. Closing as solved.
